CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### main
 
+### v7.3.0 - September 26, 2024
```

Comment:
```suggestion
```

This isn't yet released :)

Reply:
Looking at the release page of the repo the current release is 7.2.0 so won't the next release be 7.3.0? What should this be?

Reply:
I think we are supposed to put changelog entries into the `main` section. Then the release train will create a section with a proper version name and date and move pending entires there

Reply:
I changed it. Is it correct? I moved an entry from a PR I merged earlier in the week.